tests/bin/install-wc-tests.sh

```diff
@@ -10,14 +10,19 @@ DB_USER=$2
 DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
-WC_VERSION=${6-"2.6.14"}
+WC_VERSION=${6-"5.0.0"}
```

Comment:
I got confused for a second here because I didn't see that the arguments were changing based on https://github.com/Automattic/woocommerce-services/pull/2329/files#diff-52b429ab0f697095c91efe50e63752acc671f157d2661ba5404da3bdeb497b0aR55
But I see now that these are default values :)